Sources/LiveViewNative/Views/Layout Containers/Forms/Form.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Form.swift
+//  LiveViewNative
+//
+//  Created by Shadowfacts on 1/18/23.
+//
+
+import SwiftUI
+
+struct Form<R: CustomRegistry>: View {
+    @ObservedElement private var element
+    private let context: LiveContext<R>
+    
+    init(context: LiveContext<R>) {
+        self.context = context
+    }
+    
+    var body: some View {
+        SwiftUI.Form {
+            context.buildChildren(of: element)
+        }
+        .applyFormStyle(formStyle)
+    }
+    
+    private var formStyle: FormStyle {
+        switch element.attributeValue(for: "form-style") {
+        case "columns":
+            return .columns
+        case "grouped":
+            return .grouped
+        default:
+            return .automatic
+        }
+    }
+}
+
+private enum FormStyle {
```

Comment:
`ButtonStyle` and `TextFieldStyle` use a `String` raw value instead of switching manually over the cases. Could be more resilient.

```suggestion
    private var formStyle: FormStyle {
        element.attributeValue(for: "form-style").flatMap(FormStyle.init) ?? .automatic
    }
}

private enum FormStyle: String {
```